docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -59,7 +59,7 @@ image::images/alert-flyout-action-type-selection.png[UI for selecting an action
 When an alert instance matches a condition, the alert is marked as _Active_ and assigned an action group.  The actions in that group are triggered.
 When the condition is no longer detected, the alert is assigned to the _Recovered_ action group, which triggers any actions assigned to that group.
 
-**Run When** allows you to assign an action to an _action group_. This will trigger the action in accordance with your **Notify every** setting.
+**Run When** allows you to assign an action to an _action group_. This will trigger the action in accordance with your **Notify** setting.
```

Comment:
```suggestion
**Run When** allows you to assign an action to an action group. This will trigger the action in accordance with your **Notify** setting.
```